modules/rostests/apitests/ntdll/implicit_tls/implicit_tls.c

```diff
@@ -0,0 +1,55 @@
+#define WIN32_NO_STATUS
```

Comment:
```suggestion
/*
 * PROJECT:     ReactOS API Tests
 * LICENSE:     LGPL-2.0-or-later (https://spdx.org/licenses/LGPL-2.0-or-later)
 * PURPOSE:     Tests for Implicit Thread Local Storage (TLS) support
 * COPYRIGHT:   Copyright 2025 Shane Fournier <???????????????>
 */

#define WIN32_NO_STATUS
```
NOTES:
- Suggested license comes from the one used in all the other files of apitests/ntdll
- Email address to be filled.